config/pmd-test.xml

```diff
@@ -10,139 +10,114 @@
     <exclude-pattern>.*/src/main/.*</exclude-pattern>
 
     <rule ref="config/pmd.xml">
-        <!--We do throws Exception, as any exception that in not caught in test should fail a test that is expected behavior and we like it as it follow fail-fast and helps to avoid extra catch blocks-->
-        <exclude name="SignatureDeclareThrowsException"/>
-        <!--it madness to move all string values to common variables, it will be hard to read test code-->
+        <!-- Test should be named as their main class -->
+        <exclude name="AbstractNaming"/>
```

Comment:
please restore original ordering of excludes, to ease verification of what extra(not related to renaming, reordering, relocation, ....) changes you did